tests/conftest.py

```diff
@@ -132,6 +133,11 @@ def client(request):
             no_backup=setup_params["no_backup"],
         )
 
+        if client.features.model == "T":
+            apply_settings(
+                client, experimental_features=setup_params["experimental_features"]
```

Comment:
IMHO we should always enable `experimental_features` for test suite, and the test that is disabling it should explicitly disable.

same logic as moving to always enabling passphrase

Reply:
IOW let's not add this `setup_client` option at all, tests that need it should call `apply_settings` explicitly

Reply:
Fixed: 1f24bc1f0e1f5a864830453de98b638b8f84124b.